examples/asr/transcribe_speech.py

```diff
@@ -61,7 +60,6 @@
     dataset_manifest="" \
     output_filename="" \
     batch_size=32 \
-    cuda=0 \
```

Comment:
Don't remove this. Put it back as int since we need it to select the GPU at runtime for multi gpu transcribe.

Reply:
I actually recommend this and user can choose the cuda device with "CUDA_VISIBLE_DEVICES=DEVICE_ID"

Reply:
No it was decided that this remain an int. There's no need to change this.

Reply:
Could you help me understand the reason? Because we have transcribe_parallel.py for multi-gpu setting anyway

Reply:
No this is the main script and it's API was set by the ASR team so it needs to remain the same as before.

Reply:
Ok, will revert